RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/project/ProjectWizardActivity.java

```diff
@@ -119,8 +121,20 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+    private void closeKeyboardAndSearch(){
+        InputMethodManager inputManager = (InputMethodManager)
+                getSystemService(Context.INPUT_METHOD_SERVICE);
+        inputManager.hideSoftInputFromWindow((null == getCurrentFocus()) ? null : getCurrentFocus().getWindowToken(), InputMethodManager.HIDE_NOT_ALWAYS);
```

Comment:
One-liner is great if it doesn't sacrifice clarity. I believe this could be written as:

```
if (getCurrentFocus() != null) {
   inputManager.hideSoftInputFromWindow(getCurrentFocus().getWindowToken(), InputMethodManager.HIDE_NOT_ALWAYS);
}
```